docs/ascension/compatibility.mdx

```diff
@@ -0,0 +1,96 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Compatibility
+
+DiscordSRV requires Java 8 or newer
+
+## Server Software compatibility
+
+| Server software       | Supported versions                                   |
+|-----------------------|------------------------------------------------------|
+| [Bukkit], [Spigot], [Paper] | ✅ Minecraft 1.8.8 and up<br/>⚠️ Minecraft 1.7.10[^1] |
+| [BungeeCord], [Waterfall] | ✅ Minecraft 1.7.10 and up                            |
+| [Velocity]              | ✅ 3.0.0 and up<br/>❌ 2.0.0 and below                 |
+| [Sponge]                | ✅ API 8 and up<br/>❌ API 7 and below                 |
+
+[^1]: Requires [updating ASM](./faq#asm-update) <!-- TODO: fix anchor -->
```

Comment:
TODO item(s): forgotten or for later?

Reply:
Leftover from the Ascension docs. Wasn't sure if you were planning on creating a custom FAQ page or were just going to redirect the main FAQ page

Reply:
Ah, yeah that can stay there for now then